DatadogObjc/Sources/RUM/RUMDataModels+objc.swift

```diff
@@ -5544,4 +5544,4 @@ public class DDTelemetryConfigurationEventView: NSObject {
 
 // swiftlint:enable force_unwrapping
 
-// Generated from https://github.com/DataDog/rum-events-format/tree/1c476e469d5827aa1f4e60916f42ad35bbd950ef
+// Generated from https://github.com/DataDog/rum-events-format/tree/f69ca4664ed6e69c929855d02c4ce3d4b85d0bb4
```

Comment:
> ~We have an issue in webview tracking integration tests, not related to this change but rather an issue with data sent by browser sdk!~ Fixed by updating RUM models

question/ What was the mentioned problem in WebViewTracking that required change to RUM data models?

Reply:
The format changed and IntegrationTests were failing to decode events coming from the Browser SDK: https://app.bitrise.io/build/f699b944-b3eb-48a1-9c02-67e3d427af43

Reply:
Oh I see, indeed https://github.com/DataDog/rum-events-format/pull/163 introduced backward-incompatible change to RUM schema 😱.

Reply:
It's ok-ish because the SDK transmit browser events as-is without trying to decode..